utils/jwtUtil.js

```diff
@@ -92,51 +166,25 @@ function verifyToken(req, cb) {
   const token = req.session.token || req.headers.authorization;
 
   if (token) {
-    jwt.verify(token, secret, {}, (err, decodedData) => {
-      if (err) {
-        return handleInvalidToken(cb);
-      } else { // eslint-disable-line no-else-return
-        return User.findOne({ where: { name: decodedData.username } })
-        .then((user) => {
-          if (user) {
-            req.user = user; // set user in request
-
-            /*
-             * No need to check the token record if this is the default UI
-             * token.
-             */
-            if (decodedData.username === decodedData.tokenname) {
-              return cb();
-            }
-
-            return Token.findOne({
-              where: {
-                name: decodedData.tokenname,
-                createdBy: user.id,
-              },
-            })
-            .then(checkTokenRecord)
-            .then((ok) => {
-              if (ok) {
-                return cb();
-              } else { // eslint-disable-line no-else-return
-                return handleInvalidToken(cb);
-              }
-            })
-            .catch((tokenError) => handleError(cb, tokenError, 'ApiToken'));
-          } else { // eslint-disable-line no-else-return
-            return handleInvalidToken(cb);
-          }
-        });
+    verifyUserToken(req, cb)
+    .then((ret) => ret)
+    .catch((err) => {
+      if (err.explanation &&
+          err.explanation.endsWith('Refocus administrator.')) {
```

Comment:
what is the reason for checking whether the explanation endsWith Refocus administrator.?